pkg/web/templates/transactions.html

```diff
@@ -24,6 +24,7 @@ <h1 class="font-bold text-2xl">Transaction Inbox</h1>
     {{ if or (.HasRole "Admin") (.HasRole "Compliance") }}
     <div class="mt-6 flex flex-col sm:flex-row gap-y-3 sm:gap-y-0 sm:justify-between sm:items-center">
       <div class="flex items-center gap-x-1">
+        <!-- TODO: Switch anchor tag to button, update text, and add onclick event to close select msg bttn-->
```

Comment:
I'd prefer to update the button later in case we need to push a bug fix for an issue we're currently investigating.